boards/arm/mimxrt1050_evk/Kconfig.defconfig

```diff
@@ -71,8 +71,18 @@ config ETH_MCUX_0
 
 endif # NETWORKING
 
+config DISPLAY_DRIVER
```

Comment:
I think both `DISPLAY_DRIVER` and `CHARACTER_FRAMEBUFFER_SHELL_DRIVER_NAME` should be put under an if statement.

Reply:
ok